packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -553,6 +578,8 @@ class _DragTargetState<T> extends State<DragTarget<T>> {
     });
     if (widget.onAccept != null)
       widget.onAccept(avatar.data as T);
+    if (widget.onAcceptWithDetails != null)
+      widget.onAcceptWithDetails(avatar.data as T, DragTargetDetails(offset:avatar._lastOffset));
```

Comment:
I would have put the data into the DragTargetDetails. Any reason to give them special treatment keep them separate?